packages/rest/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "test-project",
+  "version": "1.0.0",
+  "description": "test",
+  "main": "index.js",
+  "author": "jan",
+  "license": "ISC",
+  "scripts": {
+    "start": "node dist/server.js",
+    "dev": "tsnd --respawn src/server.ts",
+    "build": "tsc -p .",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@aries-framework/core": "^0.1.0-alpha.266",
+    "@aries-framework/node": "^0.1.0-alpha.266",
```

Comment:
We normally specify `@aries-framework/*` as a peer Dependency (see redux-store package.json as example)